en/restore-data-using-tidb-lightning.md

```diff
@@ -6,15 +6,15 @@ aliases: ['/docs/tidb-in-kubernetes/dev/restore-data-using-tidb-lightning/']
 
 # Import Data
 
-This document describes how to import data into a TiDB cluster in Kubernetes using [TiDB Lightning](https://github.com/pingcap/tidb-lightning).
+This document describes how to import data into a TiDB cluster in Kubernetes using [TiDB Lightning](https://docs.pingcap.com/tidb/stable/tidb-lightning-overview).
 
 TiDB Lightning contains two components: tidb-lightning and tikv-importer. In Kubernetes, the tikv-importer is inside the separate Helm chart of the TiDB cluster. And tikv-importer is deployed as a `StatefulSet` with `replicas=1` while tidb-lightning is in a separate Helm chart and deployed as a `Job`.
 
-TiDB Lightning supports [three backends](https://docs.pingcap.com/tidb/stable/tidb-lightning-backends): `importer`, `local`, and `tidb`.
+TiDB Lightning supports three backends: `Importer-backend`, `Local-backend`, and `TiDB-backend`. For the differences of these backends and how to choose backends, see [TiDB Lightning Backends](https://docs.pingcap.com/tidb/stable/tidb-lightning-backends).
 
-- For the `importer` backend, both tikv-importer and tidb-lightning need to be deployed.
-- For the `local` or `tidb` backend, only tidb-lightning needs to be deployed.
-- For the `tidb` backend, it is recommended to import data using CustomResourceDefinition (CRD) in TiDB Operator v1.1 and later versions. For details, refer to [Restore Data from GCS Using TiDB Lightning](restore-from-gcs.md) or [Restore Data from S3-Compatible Storage Using TiDB Lightning](restore-from-s3.md)
+- For `Importer-backend`, both tikv-importer and tidb-lightning need to be deployed.
+- For `Local-backend` or `TiDB-backend`, only tidb-lightning needs to be deployed.
```

Comment:
How about separating the description of `Local-backend` and `TiDB-backend` here, and organizing the list like this:
- For `Importer-backend`, xxxxx
- For `Local-backend`, xxxxx
- For `TiDB-backend`, only tidb-lightning needs to be deployed and it is recommended to .....

One entry for each backend.